.ci/setup.sh

```diff
@@ -38,6 +38,12 @@ if [ -n "$PULL_REQUEST_NUMBER" ] || [ -n "$LOCALCI_PR_NUMBER" ]; then
 	bash -f "${cidir}/run_fetch_branches_tool.sh"
 fi
 
+echo "Install Clear containers image"
+"${cidir}/install_asset.sh" "image" "latest"
```

Comment:
Hi @jcvenegas 
Sorry for posting a post-merge follow up. I think this has broken the metrics Jenkins CI instance.
The root problem I think is that on the metrics CI, which is a bare metal system (as opposed to the VM Azure systems we use QA CI) - so we have clear containers installed into docker as the default runtime from the previous PR test (we don't clean/reset the whole system like we effectively do with Azure - where we get a new clean VM machine for each PR).

This results in the known build failure for osbuilder (like https://github.com/clearcontainers/osbuilder/issues/8), like:

```
17:26:17 Step 2/9 : RUN dnf install -y qemu-img parted gdisk make gcc bc git e2fsprogs libudev-devel pkgconfig
17:26:17  ---> Running in 6220fc9fd574
17:26:25 [91mBDB0126 mmap: Invalid argument
17:26:25 [0m[91mhandle_proxy_response:616:Error response received from proxy at /run/virtcontainers/pods/6220fc9fd574d56d747fe5bcd635ae18ab67dc190fca8e4b97b6a9dda0e08d2f/proxy.sock: {"msg":"vm: unknown token 7yumQLpbAES-kAnPNgsZZswCu_WH7hkFBnhpKa-bMqg="}
17:26:25 
17:26:25 /usr/libexec/clear-containers/cc-shim: Shim received an error in responseto ConnectShim command, exiting[0mThe command '/bin/sh -c dnf install -y qemu-img parted gdisk make gcc bc git e2fsprogs libudev-devel pkgconfig' returned a non-zero code: 1
17:26:25 Makefile:114: recipe for target 'docker-build' failed
```
from https://clearlinux.org/cc-ci/job/clear-containers-runtime-16.04-PR/66/console

As the metrics CI system runs Ubuntu right now, I suspect we suffer from a combination of https://github.com/clearcontainers/osbuilder/issues/8 and https://github.com/clearcontainers/osbuilder/issues/22.

I can't think of a workaround or solution right now. The only thing that comes to mind is maybe for a system that has CC as the default runtime we will need to use the latest pre-built image or kernel from the repos rather than trying to build the latest from source.

Thoughts?